### PR TITLE
feat: add version endpoint to api server

### PR DESCRIPTION
**Pull request by dbcfd**

### CODE REVIEW

**Review by oed — COMMENTED** [review]
Looks like we are using "node" in some places and "server" in others. Let's align.

File: api/build.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let mut opts = built::Options::default();
+    opts.set_dependencies(true);
+
+    let src = project_root::get_project_root().unwrap();
```
Comment (AaronGoldman) [inline api/build.rs:5]: What is saving this data for?

**Comment by AaronGoldman** [comment]
LGTM

File: api/ceramic.yaml [hunk 2]
```diff
@@ -79,8 +90,18 @@ components:
...
+    Version:
+      title: Version
+      description: Version of the ceramic server
+      type: object
+      required:
+        - version
+      properties:
+        version:
+          type: string
+          description: Version of the ceramic server
```
Comment (nathanielc) [inline api/ceramic.yaml:102]: Are we making any guarantees as to the content of this version string? Is it semver? Something else. Can they be compared in any way?

**Review by nathanielc — APPROVED** [review]
LGTM with one question about the API spec.

File: api/ceramic.yaml (hunk 2 above)
Comment (dbcfd) [reply api/ceramic.yaml:102]: It is semver. Do we want to define a semver type, or just say the string is semver in description?

Comment (nathanielc) [reply api/ceramic.yaml:102]: Semver in the description is good enough for me.

**Review by nathanielc — APPROVED** [review]

**Review by smrz2001 — APPROVED** [review]
🚀

**Pull request merged**